libraries/AP_Scripting/docs/docs.lua

```diff
@@ -3761,6 +3761,10 @@ AC_AttitudeControl = {}
 ---@return number -- yaw slew rate
 function AC_AttitudeControl:get_rpy_srate() end
 
+-- return current VTOL total attitude error in degrees
```

Comment:
This is only roll and pitch, I wonder if we should just copy the function description exactly from attitude control.  That would be:

```
Return the angle between the target thrust vector and the current thrust vector in degrees.
```

Reply:
changed